manager/src/main/java/com/redhat/service/bridge/manager/api/internal/ShardBridgesSyncAPI.java

```diff
@@ -29,6 +30,7 @@
 @Path(APIConstants.SHARD_API_BASE_PATH)
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
+@Transactional
```

Comment:
The idea was to move `@Transactional` to the entry points as quarkus suggests https://quarkus.io/guides/transaction . Anyway, since I've moved to `join fetch` I've reverted the changes